LocalMultiplayerAgent/BuildTool/readme.md

```diff
@@ -0,0 +1,114 @@
+Technically, you can just run game server deployment without testing with LMA **but testing is highly recommended**
+
+## Requirements:
+- Set your [PlayFab Title Secret Key](https://docs.microsoft.com/en-us/gaming/playfab/gamemanager/secret-key-management) as a **User Environment Variable**
+    - Command Prompt:
+    `setx PF_SECRET "[variable value]"`
+
+    - Powershell:
+    `[Environment]::SetEnvironmentVariable("PF_SECRET","[variable value]","User")`
+
+- Make sure your [MultiplayerSettings.json](../MultiplayerSettings.json) is configured right
+    - [Windows Process](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/localmultiplayeragent/run-process-based-gameserver):
+        ```json
+        "RunContainer": false
+        "ProcessStartParameters": {
+            "StartGameCommand": "<your_game_server_exe>"
+        }
+        ``` 
+    - [Windows Container](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/localmultiplayeragent/run-container-gameserver):
+        ```json
+        "RunContainer": true
+        "ContainerStartParameters": {
+            "StartGameCommand": "C:\\Assets\\<your_game_server_exe>",
+            "ResourceLimits": {
+                "Cpus": 0,
+                "MemoryGib": 0
+            },
+            "ImageDetails": {
+                "Registry": "mcr.microsoft.com",
+                "ImageName": "playfab/multiplayer",
+                "ImageTag": "wsc-10.0.17763.2458",
+                "Username": "",
+                "Password": ""
+             }
+        }
+        ```
+    - [Linux Container on Windows](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/localmultiplayeragent/run-container-gameserver):
+        ```json
+        "RunContainer": true
+        "ContainerStartParameters": {
+            "StartGameCommand": "C:\\Assets\\<your_game_server_exe>",
+            "ResourceLimits": {
+                "Cpus": 0,
+                "MemoryGib": 0
+            },
+            "ImageDetails": {
+                "Registry": "mydockerregistry.io",
+                "ImageName": "mygame",
+                "ImageTag": "0.1",
+                "Username": "",
+                "Password": ""
+             }
+        }
+        ```
+
+    - These parameters in MultiplayerSettings.json are not required if you are only trying to create a build:
+    ```json  
+    "OutputFolder": "",
+    "NumHeartBeatsForActivateResponse": 5,
+    "NumHeartBeatsForTerminateResponse": 10,
+    "AgentListeningPort": 56001,
+    "BuildId": "",
+    "Region": "",
+    "SessionConfig": {
+        "SessionId": "ba67d671-512a-4e7d-a38c-2329ce181946",
+        "SessionCookie": null,
+        "InitialPlayers": [ "Player1", "Player2" ]
+    }
+    ```
+
+    - Linux Process is currently not supported on Local Multiplayer Agent, so users cannot create a build with this server type
```

Comment:
can we add that users cannot create a Linux process build with "this build tool"?  it sounds like we don't support Linux process build